### PR TITLE
Spell check fixes for sprint #60

### PR DESCRIPTION
**Pull request by clebergnu**
Signed-off-by: Cleber Rosa <crosa@redhat.com>

### CODE REVIEW

File: examples/tests/doublefree.py [hunk 1]
```diff
@@ -17,7 +17,8 @@ class DoubleFreeTest(Test):
 
     :avocado: tags=requires_c_compiler
 
-    :param source: name of the source file located in deps path
+    :param source: name of the source file located in a the data
```
Comment (apahim) [inline examples/tests/doublefree.py]: s/in a the/in the/

File: examples/tests/doublefree_nasty.py [hunk 2]
```diff
@@ -16,7 +16,8 @@ class DoubleFreeTest(Test):
 
     :avocado: tags=failure_expected,requires_c_compiler
 
-    :param source: name of the source file located in deps path
+    :param source: name of the source file located in a the data
```
Comment (apahim) [inline examples/tests/doublefree_nasty.py]: ditto

File: examples/tests/modify_variable.py [hunk 3]
```diff
@@ -19,7 +19,8 @@ class PrintVariableTest(Test):
 
     :avocado: tags=requires_c_compiler
 
-    :param source: path to the source file relative to deps dir.
+    :param source: name of the source file located in a the data
```
Comment (apahim) [inline examples/tests/modify_variable.py]: also here

File: examples/tests/multiple_tests.py [hunk 4]
```diff
@@ -6,10 +6,9 @@
 class MultipleTests(Test):
 
     """
-    Following the idea of unittest module,
-    every test method starts with a 'test' prefix,
-    so that 'test_foo' and 'testFoo' are test methods,
-    but 'division_by_zero' and 'action' are not.
+    Following the unittest module pattern, every test method starts
+    with a literal 'test' prefix, so that 'test_foo' and 'testFoo' are
+    test methods, but 'division_by_zero' and 'action' are not.
```
Comment (apahim) [inline examples/tests/multiple_tests.py]: ok

File: examples/tests/synctest.py [hunk 5]
```diff
@@ -14,7 +14,7 @@ class SyncTest(Test):
     """
     Execute the synctest test suite.
 
-    :param sync_tarball: path to the tarball relative to deps dir.
+    :param sync_tarball: path to the tarball relative to a data directory
```
Comment (apahim) [inline examples/tests/synctest.py]: ok

File: spell.ignore [hunk 6]
```diff
@@ -564,3 +564,17 @@ virtuozzo
...
+workdir
+datasize
+getoutput
+doublefree
+synctest
+getstatusoutput
+rtd
+testfoo
+commandtimeout
+uids
```
Comment (apahim) [inline spell.ignore]: ok

**Comment by clebergnu** [comment]
True, good catch.

**Comment by clebergnu** [comment]
Closing in favor of #2553

**Pull request closed without merging**